app/models/room.rb

```diff
@@ -5,12 +5,12 @@ class Room < ApplicationRecord # :nodoc:
   has_many :reservations, dependent: :destroy
 
   # Enums
-  enum room_type: %i[single double triple family_suite]
+  enum category: %i[single double triple family_suite]
```

Comment:
I think `type` would be easier to understand, but I am also not sure if `category` is a more common name in the hotel jargon.

Reply:
Category is more usual on hotel industry, let's keep like this for now. : )